### ISSUE TITLE
Doubt about an answer marked as right

### ISSUE THREAD

**Issue by l-vo** [certificationy/symfony-pack#76]
Are you sure for this one ?
https://github.com/certificationy/symfony-pack/blob/bb5c67986573b52e574c7938c2c46fdc7c746fdc/data/command-line.yml#L147

Event dispatching seems not be deactivable:
https://github.com/symfony/symfony/blob/4d21ab4e7bbc1578c7f23b65f56fa605aff4fb10/src/Symfony/Component/Console/Application.php#L916
In the code above, there is no condition checking if events are disabled.

**Comment by fliespl** [comment]
Only two years have passed...

Yes, answer is correct :)

```
            $this->dispatcher->dispatch(ConsoleEvents::COMMAND, $event);

            if ($event->commandShouldRun()) {
                $exitCode = $command->run($input, $output);
            } else {
                $exitCode = ConsoleCommandEvent::RETURN_CODE_DISABLED;
            }
```

as you can see - there is even return code for it.

**Comment by l-vo** [comment]
@a-menshchikov the answer is about the possibility to disable console events, not the possibility to disable console commands.

**Comment by a-menshchikov** [comment]
Oh, you are right!
It seems to me as a misprint.

**Comment by l-vo** [comment]
@a-menshchikov maybe 🙂

**Comment by mickaelandrieu** [comment]
Yes, you're right : feel free to contribute the deck to make this answer right please 👼 

Regards,

**Comment by a-menshchikov** [comment]
@l-vo this issue seems as outdated. Maybe it should be closed?

**Comment by l-vo** [comment]
Fixed by #87
